docs/headlampFAQ.md

```diff
@@ -0,0 +1,132 @@
+---
+Title: Frequently Asked Questions
+---
+
+# General Questions
+
+### Q: What is Headlamp and who is it for?
+
+**A:** Headlamp is an open source, extensible web interface designed to simplify the management of Kubernetes clusters. It serves as a visual management tool ideal for developers, DevOps professionals, and system administrators who are looking for an intuitive way to oversee cluster operations, deployments, and troubleshooting.
+
+### Q: On which platforms can I run Headlamp?

```

Comment:
This question should be split in two:
Q: What Kubernetes flavors or vendors does Headlamp support?

Q: Is Headlamp a desktop or web application?